api/chat.go

```diff
@@ -68,6 +76,106 @@ func configGinChatRouter(router *gin.RouterGroup) {
 	})
 }
 
+func handleSubmitPollOptionVote(ctx tools.TUMLiveContext, msg []byte) {
+	var req submitPollOptionVote
+	if err := json.Unmarshal(msg, &req); err != nil {
+		log.WithError(err).Warn("could not unmarshal submit poll answer request")
+		return
+	}
+	if ctx.User == nil {
+		return
+	}
+
+	if err := dao.AddChatPollOptionVote(req.PollOptionId, ctx.User.ID); err != nil {
+		return
+	}
+
+	voteCount, _ := dao.GetPollOptionVoteCount(req.PollOptionId)
+
+	voteUpdateMap := gin.H{
+		"pollOptionId": req.PollOptionId,
+		"votes":        voteCount,
+	}
+
+	if voteUpdateJson, err := json.Marshal(voteUpdateMap); err == nil {
+		broadcastStream(ctx.Stream.ID, voteUpdateJson)
+	}
+}
+
+func handleStartPoll(ctx tools.TUMLiveContext, msg []byte) {
+	var req startPollReq
```

Comment:
if you only use the type once you can declare it here in the function scope (nit)

Reply:
yes nicer, but a little inconsistent with the other req types. But probably It's better to continue this way :)